frontend/src/components/Comment/Comment.js

```diff
@@ -1,21 +1,14 @@
 import React from 'react'
-import "./Comment.scss";
-import {Link} from "react-router-dom";
-
-class Comment extends React.Component {
-    //todo: add like on comments
-    render() {
-        return (
-            <div className="comment-div">
-                <br/>
-                <Link to={`/account/${this.props.comment.account}`}>{this.props.comment.account}</Link>: {this.props.comment.comment}
-                <div className="date-created">{new Date(this.props.comment.created).toUTCString()}</div>
-                <br/>
-            </div>
-        )
-    }
-}
-
-export default Comment;
-
-
+import { Link } from "react-router-dom"
+import { useEffect } from 'react'
```

Comment:
Not sure this import is necessary since I don't see you use `useEffect` anywhere?